mayday/file.go

```diff
@@ -3,11 +3,42 @@ package mayday
 import (
 	"archive/tar"
 	"io"
+	"log"
 )
 
 type File struct {
```

Comment:
Could we rename this to something a little less general? I find myself thinking "oh, this is a standard `File` typed var... no wait, it's the special Mayday file-like type"

Change to something like `MaydayFile` or `IngestFile`, etc?